docs/en/observability/tab-widgets/uptime-monitoring/run-test-configure-content.asciidoc

```diff
@@ -85,6 +85,9 @@ Where to deploy the monitor. Monitors can be deployed in multiple locations so t
 Control the monitor's download speeds, upload speeds, and latency to simulate your application's behavior on slower or laggier networks.
 `screenshot` (https://github.com/elastic/synthetics/blob/{synthetics_version}/src/common_types.ts#L190[`ScreenshotOptions`])::
 Control whether or not to capture screenshots. Options include `'on'`, `'off'`, or `'only-on-failure'`.
+`privateLocations` (`Array<string>`)::
+The <<private-locations,private locations>> to which the monitors will be deployed. These private locations refer to locations hosted and managed by you, whereas 
+<<global-managed-testing-infrastructure,`locations`>> are hosted by Elastic.
```

Comment:
```suggestion
<<global-managed-testing-infrastructure,`locations`>> are hosted by Elastic. You can specify a private location by the location's name. 
```

just want to call out how to specify the private location, since private locations sorta-kinda use an "id" whereas users can specify their public locations by their configured location name.